doCNA/Distribution.py

```diff
@@ -22,39 +22,64 @@ def __init__ (self, values, p_thr = 0.3, thr_z = 1.0):
 
         """
         assert len (values) > Consts.LENGTH_THRESHOLD, print ("Not enough data points to consider distribution.")
-        single_G_par = fit_single_G (np.sort(values), alpha = 0.01, r = 0.5)
+        
+        try: 
+            single_G_par = fit_single_G (np.sort(values), alpha = 0.01, r = 0.5)
+            values_used = 'all'
+        except:
```

Comment:
when does this happen? why would this raise an exception?

Reply:
sometimes there many identical values. I could not think of better solution here.